packages/next-on-pages/docs/supported.md

```diff
@@ -6,13 +6,11 @@
 
 ## Supported Next.js versions
 
-Based on manual testing, it seems like the latest Next.js version (`13.4.2` at the time of writing) is supported.
+`@cloudflare/next-on-pages` supports all minor and patch version of Next.js 13. We regularly run manual and automated tests to ensure such compatibility.
 
-Earlier and Later versions might be only partially supported, we don't fully know.
+Next.js canary versions are not included.
 
-> The team is very soon going to test various different versions and make improvements to the documentation on which Next.js versions are supported and to what degree.
-
-## Supported Features
+> The edge runtime has been introduced as experimental since [Next.js 12.2](https://nextjs.org/blog/next-12-2#edge-server-rendering-experimental) so versions prior that that are inherently not compatible with the Cloudflare Pages runtime. Moreover the Next.js edge runtime has been significantly experimental early on, that's why we don't recommend using any Next.js version earlier than 13.0.0.
```

Comment:
Do we need to include this paragraph? I think the previous paragraphs seems sufficient to me. Let's be clear about what we support, and anything we don't mentioned is by default unsupported.

Reply:
no I don't think it's strictly needed, I just thought of sort of providing the reason as to why we don't support earlier Next.js versions, but yeah if you think that this just adds extra fluff (and I can see why) I don't mind removing it 🙂👍